tests/integration/test_keeper_reconfig_remove/test.py

```diff
@@ -42,46 +44,43 @@ def test_reconfig_remove_followers_from_3(started_cluster):
     Check that remaining node is in standalone mode.
     """
 
-    zk1 = get_fake_zk(node1)
-    config, _ = zk1.get("/keeper/config")
-    config = config.decode("utf-8")
+    global zk1, zk2, zk3
```

Comment:
why?

Reply:
To have a graceful shutdown https://github.com/ClickHouse/ClickHouse/pull/54201/files#diff-dbf5606f605734e2f5d6335548b5f91e728fb9084255e3d397dee333ac792679R34